cfme/tests/containers/test_containers_overview_data_integrity.py

```diff
@@ -81,15 +75,42 @@ def test_containers_overview_data_integrity():
     navigate_to(ContainersOverview, 'All')
     # We should wait ~2 seconds for the StatusBox population
     # (until we find a better solution)
+    # Since we collect images from Openshift and from the pods,
+    # images are tested separately
     time.sleep(2)
     statusbox_values = {data_set.object: int(StatusBox(data_set.name).value())
                         for data_set in DATA_SETS}
-    api_values = get_api_object_counts(list_providers_by_class(ContainersProvider))
+    api_values = get_api_object_counts(
+        list_providers_by_class(ContainersProvider))
+
+    list_img_from_registry = provider.mgmt.list_image()
+    list_img_from_registry_splitted = [i.id.split(
+        '@sha256:')[-1] for i in list_img_from_registry]
+
+    list_img_from_openshift = provider.mgmt.list_image_openshift()
+    list_img_from_openshift_splitted = [d['name']
+                                        for d in list_img_from_openshift]
+    list_img_from_openshift_parsed = [i[7:].split(
+        '@sha256:')[-1] for i in list_img_from_openshift_splitted]
+
+    for s in list_img_from_openshift_parsed:
+        for item in list_img_from_registry_splitted:
+            if item not in list_img_from_openshift_parsed:
+                list_img_from_openshift_parsed.append(item)
```

Comment:
Why are you iterating over list_img_from_openshift_parsed if you don't use 's'? it causes to that you're appending every 'item' to list_img_from_openshift_parsed many times instead of once.
also, you have already defined list_img_from_openshift_parsed in line 93 so why are you do it again (with "not splitted" items - do we need them splitted or not?)?
anyway, I'd suggest to extend it with list comprehension:
`list_img_from_openshift_parsed.extend([item for item in list_img_from_registry_splitted if item not in list_img_from_openshift_parsed])`